config/skim_UL16APV.cfg

```diff
@@ -0,0 +1,126 @@
+[selections]
+# Store only ETau/MuTau/TauTau/MuMu events in the skims
+onlyFinalChannels = true
+
+[parameters]
+home = /grid_mnt/vol_home/llr/cms/portales/hhbbtautau/KLUB_UL_20230830/CMSSW_11_1_9/src/
+# for 2016, bits are inverted for the loose wp, https://twiki.cern.ch/twiki/bin/view/CMS/PileupJetIDUL#Recommendations_for_2016_UL_data
+PUjetIDWP           = 0 # 2:tight - 1:medium - 0:loose
+PFjetIDWP           = 1 # 0:don't pass - 1:tight - 2:tightLeptonVeto
+# if false, SS events are saved
+# saveOS = 0 : save the same sign events only
+# saveOS = 1 : save the opposite sign events only
+# saveOS = -1 : save both SS and OS events
+saveOS              = -1
+# minimal distance required between any jets and the two leptons of the event
+# when counting the jets in the jet selection and when choosing the two bjet candidates
+# additional jets do NOT undergo this selection
+lepCleaningCone     = 0.5
+# choose the algo used to ID the b-jets
+# 1 = the two jets surviving selections with largest btag value
+# 2 = the two jets surviving selections with largest pT
+bChoiceFlag         = 1
+# choose the MC to be reweighted 
+PUReweightMC        = 2016
```

Comment:
Same as for UL16?

Reply:
This is a very old flag which I honestly barely understand. Reading here https://github.com/LLRCMS/KLUBAnalysis/blob/86b10a26673a46e4ec1d26651141a03bd774baae/src/PUReweight.cc#L79 it seems to me that either 2016 and 2018 are file: the actual reweighing will be done using the txt file. @kramerto could please check what happens for 2017? Do you have a meaningful PU distribution?

Reply:
Answering to myself, sorry, 2017 is also fine: https://github.com/LLRCMS/KLUBAnalysis/blob/86b10a26673a46e4ec1d26651141a03bd774baae/src/PUReweight.cc#L69